<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMainWindow</class>
 <widget class="QMainWindow" name="TMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №3 (by IVAN_IV)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="editLog">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLog">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Лог</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadFromFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить из файла</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblHeuristics">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Эвристики</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbNoHeuristic">
       <property name="text">
        <string>Нет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbManhattanHeuristic">
       <property name="text">
        <string>Манхэттэна</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnEvklidHeuristic">
       <property name="text">
        <string>Эвклида</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbChebyshevaHeuristic">
       <property name="text">
        <string>Чебышева</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnFindPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти путь</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbUseTimer">
       <property name="text">
        <string>Замер времени</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbVisualize">
       <property name="text">
        <string>Визуализация</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblStartPoint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начальная координата</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndPoint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Конечная координата</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spboxStartX">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinboxEndX">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spboxStartY">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinboxEndY">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>630</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblXCoord">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblYCoord">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
